src/libYARP_dev/include/yarp/dev/IJoypadController.h

```diff
@@ -139,7 +143,7 @@ class  YARP_dev_API yarp::dev::IJoypadController
     /**
      Get the value of a touch if present, return false otherwise
     * @brief getTouch
-    * @param value a vector that will contain the joystick position. the size of the vector will give you the Number of degrees of freedom
+    * @param value a vector that will contain the touch values normalized from 0.0 to 1.0.
```

Comment:
I tested this on a real joypad, and the "touch" values seemd to be xy coordinates, is this correct?